From 305bf2801a56acad229fbc276ffdff90ab498549 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Ball=C3=B3=20Gy=C3=B6rgy?= <ballogyor@gmail.com>
Date: Sun, 18 Jan 2026 00:00:18 +0100
Subject: [PATCH] Exclude more environments

These desktop environments have their own tools to configure the appearance.
---
 data/lxappearance.desktop.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/data/lxappearance.desktop.in b/data/lxappearance.desktop.in
index 54fc0a2..14a7774 100644
--- a/data/lxappearance.desktop.in
+++ b/data/lxappearance.desktop.in
@@ -6,6 +6,6 @@ _Comment=Customizes look and feel of your desktop and applications
 _Keywords=windows;preferences;settings;theme;style;appearance;
 Icon=preferences-desktop-theme
 Exec=lxappearance
-NotShowIn=GNOME;KDE;XFCE;MATE;
+NotShowIn=GNOME;KDE;XFCE;MATE;Budgie;COSMIC;Enlightenment;LXQt;X-Cinnamon;
 StartupNotify=true
 Categories=GTK;Settings;DesktopSettings;X-LXDE-Settings;
